sign_verify_file_ssh.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+# Usage message
+usage() {
+    echo "Usage: $0 sign <file> <private_key> | verify <file> <public_key> [signature_file]"
```

Comment:
Do we need verification here?

Reply:
Usage info is very short. Maybe some information about the supported format of they private key would be good.